### PR TITLE
Added new column in configuration table

### PR DESCRIPTION
**Pull request by shrutimandaliya**

### CODE REVIEW

**Comment by shrutimandaliya** [comment]
solve issue #724

File: src/main/resources/alfio/db/PGSQL/V19_1.11.0__EXPAND_CONFIGURATION_VALUE.sql [hunk 1]
```diff
@@ -16,6 +16,7 @@
 --
 
 alter table configuration alter column c_value SET DATA TYPE text;
+alter table configuration add column last_modified_date timestamp with time zone;
```
Comment (syjer) [inline src/main/resources/alfio/db/PGSQL/V19_1.11.0__EXPAND_CONFIGURATION_VALUE.sql:19]: you don't modify an existing sql migration file, additionally, the value last modified date is not set, so adding only the column is not helpful

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
 server.forward-headers-strategy=native
 server.servlet.session.cookie.http-only=true
 server.servlet.session.timeout=2h
-server.compression.enabled=true
\ No newline at end of file
+server.compression.enabled=true
```
Comment (syjer) [inline src/main/resources/application.properties:4]: don't reformat file, revert change.

File: src/main/java/alfio/model/system/Configuration.java [hunk 3]
```diff
@@ -24,6 +24,7 @@
 import org.apache.commons.lang3.builder.EqualsBuilder;
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 
+import java.util.Date;
```
Comment (syjer) [inline src/main/java/alfio/model/system/Configuration.java:27]: unused import.

**Comment by shrutimandaliya** [comment]
If i can not change SQL configuration files by query then how i can add new column,can you suggest me ,is there any alternative way to add a new column?

**Comment by syjer** [comment]
hi @shrutimandaliya , sorry but I'll close this PR. It's clear you are not understanding what you are doing and I can't really help you here, it's costing too much of my time and attention.

I hope you understand.

**Pull request closed without merging**